server/agent/models/conversation.py

```diff
@@ -18,3 +22,15 @@ class Meta:
             "are mostly questions and answers and have different actors such as end user asking "
             "question and ECL agent answering those questions."
         )
+
+
+class ConversationFile(models.Model):
+    conversation = models.ForeignKey(Conversation, on_delete=models.PROTECT, related_name="files")
```

Comment:
A general question - do we end up uploading files to a conversation, or to a particular message inside that conversation?

Reply:
Finally it will be a message. Because we can still get all file for conversation via message foregin key and also there will be kinda file message type to display uploaded files in UI

Reply:
Alright - do you later switch this foreign key from conversation to message (given that you can then get conversation by joining with messages)?